src/index.html

```diff
@@ -11,6 +11,24 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="search search--big" data-qa="big">
+      <div class="search--bigimage"></div>
+      <input type="text"
+      class="key key--big"
+      placeholder="Try “Los Angeles”"
+      data-qa="keypress"
+      >
```

Comment:
Need to fix code style everywhere
```suggestion
      <input
        type="text"
        class="key key--big"
        placeholder="Try “Los Angeles”"
        data-qa="keypress"
      >
```